deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/layers/mkldnn/MKLDNNLSTMHelper.java

```diff
@@ -43,6 +44,8 @@
 import java.util.Map;
 
 public class MKLDNNLSTMHelper implements LSTMHelper {
+    public MKLDNNLSTMHelper(DataType dataType) {}
+
```

Comment:
default empty ctor will not be generated if I remember correctly. is it intentional?